### DISCUSSION TITLE
Flags... Do you fly them? Other recondition a

### DISCUSSION THREAD

**Discussion by PovilasID** [blakeblackshear/frigate#9891] [General]
Sorry... this is bit of rant but today I am proud/legally obligated to fly a flag on my house. 
Frigate is going insane... 9 times out of 10 it thinks it's a person and other times... it's a bike! And it is recognizing a new person every 5-10 minutes.
I am using OpenVINO and on 0.13.1 Frigate version

a) Will Frigate+ help reduce these?
b) Have you noticed any 'Achilles heals' in your systems that just brake Frigate?

**Comment by NickM-27** [comment]
1. It should but that's not required. You could use an object mask to block these out 
2. I personally don't
